test/ItspServices.pServer.ClientTest/ServerCommunicatorTests.cs

```diff
@@ -56,14 +43,16 @@ public async Task ServerCommunicator_RequestRootFolder_ShouldReturnRootFolder()
                 ProtectedDataIds = new List<int>(),
                 SubfolderIds = new List<int>()
             };
-
-            MockHttpMessageHandler.Callback = (request) =>
+            Mock<IHttpClientFactory> clientFactory = new Mock<IHttpClientFactory>();
+            Func<HttpRequestMessage, HttpResponseMessage> callback = (request) =>
```

Comment:
This can become a local function.
See [here](https://www.davidhayden.me/blog/csharp-local-functions-visual-studio-2017)